cpp/cmake/thirdparty/get_cutlass.cmake

```diff
@@ -79,14 +82,15 @@ function(find_and_configure_cutlass)
   )
 endfunction()
 
+set(_cutlass_version 3.5.1)
```

Comment:
If we get an *installed* version it's not a problem, the patch only fixes the build directory export, which won't be present in an install.

Reply:
Also, isn't it already going to download anyway, since `PATCH_COMMAND` was passed?

https://github.com/rapidsai/rapids-cmake/blob/6cbdb0363b3be9681031d52e0da7cee743450c01/rapids-cmake/cpm/find.cmake#L186-L190

Reply:
Yall are correct. Looks like I don't remember all the ways we made sure that `rapids_cpm_find` works as expected when a patch is provided